bazel/scripts/run_integ_tests.sh

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -euo pipefail
+
+###############################################################################
+# FUNCTION DECLARATIONS
+###############################################################################
+
+help() {
+    echo "Executes all integration tests."
+    echo "Usage:"
+    echo "   $(basename "$0")" 
+    echo "      Execute all integration tests in the magma repository." 
+    echo "   $(basename "$0") path_to_tests_directory:bazel_test_target_name"
+    echo "      Execute the specified test."
+    echo "   $(basename "$0") --list"
+    echo "      List all integration tests."
+    echo "   $(basename "$0") --list-extended"
+    echo "      List all tests that use the traffic server."
+    echo "   $(basename "$0") --list-traffic-server"
+    echo "      List the extended integration tests."
+    echo "   $(basename "$0") --setup-extended"
+    echo "      Execute the setup test for the extended tests."
+    echo "   $(basename "$0") --teardown-extended"
+    echo "      Execute the teardown test for the extended tests."
+    echo "   $(basename "$0") --skip-setup-teardown-extended"
+    echo "      Execute all tests in the magma repository," 
```

Comment:
Perhaps we can insert another "integration" in this line?

Reply:
Fixed.